product_configurator/models/product_attribute.py

```diff
@@ -100,6 +101,19 @@ def check_searchable_field(self):
                   self.custom_type)
             )
 
+    def validate_custom_val(self, val):
+        """ Pass in a desired custom value and ensure it is valid.
+        Probaly should check type, etc, but let's assume fine for the moment.
+        """ 
+        self.ensure_one()
+        if self.custom_type in ('int', 'float'):
+            if self.min_val or self.max_val:
+                val = literal_eval(val)
+                if val < self.min_val or val > self.max_val:
```

Comment:
Wouldn't this fail if max_val is not defined (meaning it has no max value) and it would default to 0?